apps/acrs/translations/src/en/validation.json

```diff
@@ -193,6 +193,9 @@
     "has-family-in-uk": {
       "required": "You must select an option"
     },
+    "has-family-in-uk": {
```

Comment:
There's a duplicate configuration for `has-family-in-uk`.

Reply:
yes that has been removed